dietpi/dietpi-software

```diff
@@ -5519,6 +5529,36 @@ Package: wireguard wireguard-dkms wireguard-tools\nPin: release n=bullseye\nPin-
 			G_EXEC cd /tmp/$G_PROGRAM_NAME
 
 		fi
+		
+		software_id=183 # Bitwarden RS
+                if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+                    Banner_Installing
+
+                    # Dependencies: https://github.com/dani-garcia/bitwarden_rs/wiki/Building-binary#dependencies
+                    DEPS_LIST='pkg-config libssl-dev'
+
+                    # Rust from https://rustup.rs/
+                    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain none
+                    source "$HOME/.cargo/env"
```

Comment:
Don't use `$HOME` here, use `/root` instead to be explicit and failsafe. The script does forcefully run as root user anyway.

Reply:
Makes sense